src/xy/xyRolling.js

```diff
@@ -0,0 +1,24 @@
+import { xRolling } from '../x/xRolling';
+/**
+ * This function calculates a rolling average
+ * @param {ArrayPoints} [points] array of points {x,y}
+ * @param {object} [options={}]
+ * @param {number} [options.window=5] rolling window
+ * @param {function} [fct] callback function that from an array returns a value.
+ * @param {string} [options.padding.size=0] none, value, circular, duplicate
+ * @param {string} [options.padding.algorithm='value'] none, value, circular, duplicate
+ * @param {number} [options.padding.value=0] value to use for padding (if algorithm='value')
```

Comment:
An additional feature one could think about is to also accept an array as the Python implementation does (https://numpy.org/doc/stable/reference/generated/numpy.pad.html)